doc-content/enterprise-only/openshift/architecture-authoring-con.adoc

```diff
@@ -87,7 +87,7 @@ The controller strategy is not suitable for clustered deployment. In an OpenShif
 
 Each {KIE_SERVER} deployment (which can be scaled) creates a ConfigMap that reflects its current state. The {CENTRAL} discovers all {KIE_SERVERS} by reading their ConfigMaps.
 
-When the user requests a change in the {KIE_SERVER} configuration (for example, deploys or undeploys a service), {CENTRAL} initiates a connection to {KIE_SERVER} and sends a REST API request. {KIE_SERVER} changes the ConfigMap to reflect the new configuration state and then triggers its own redeployment, so that all instances are redeployed and reflect the new configuration.
+When the user requests a change in the {KIE_SERVER} configuration (for example, deploys or undeploys a service), {CENTRAL} initiates a connection to {KIE_SERVER} and sends a REST API request. {KIE_SERVER} changes the ConfigMap to reflect the new configuration state and then triggers its own redeployment, so that all instances are redeployed and reflect the new configuration. For more information about ConfigMaps, see {URL_DEPLOYING_ON_OPENSHIFT}#configmaps-con[_KIE configuration and ConfigMaps_].
```

Comment:
Own may be redundant. 
```suggestion
When the user requests a change in the {KIE_SERVER} configuration (for example, deploys or undeploys a service), {CENTRAL} initiates a connection to {KIE_SERVER} and sends a REST API request. {KIE_SERVER} changes the ConfigMap to reflect the new configuration state and then triggers its redeployment, so that all instances are redeployed and reflect the new configuration. For more information about ConfigMaps, see {URL_DEPLOYING_ON_OPENSHIFT}#configmaps-con[_KIE configuration and ConfigMaps_].
```

Reply:
Removed